Formula/docker-clean.rb

```diff
@@ -0,0 +1,14 @@
+class DockerClean < Formula
+  desc "Script cleans Docker containers, images, networks, and volumes."
```

Comment:
Please tweak the description and omit the full stop at the end. I think the following is better and more concise:

```ruby
desc "Clean Docker containers, images, networks, and volumes"
```